app/helpers/svg_helper.rb

```diff
@@ -0,0 +1,7 @@
+module SvgHelper
```

Comment:
I like this, but there is a gem that could be useful in the future too.

https://github.com/jamesmartin/inline_svg

Sorry I didn't comment earlier